src/npctalk_funcs.cpp

```diff
@@ -587,6 +587,8 @@ void talk_function::give_equipment_allowance( npc &p, int allowance )
 {
     std::vector<item_pricing> giving = npc_trading::init_selling( p );
     int chosen = -1;
+    // Add asked_for_item effect whether or not NPC has items to give
+    p.add_effect( effect_asked_for_item, 3_hours );
```

Comment:
```suggestion
    p.add_effect( effect_asked_for_item, 3_hours );
```
3 days or even a week seems more reasonable than able to ask every three hours?

Reply:
Yes will change it to 3 days once the other conversation is resolved.

Reply:
@Maleclypse the conversation has been resolved if all else looks good please merge.